microprot/scripts/split_search.py

```diff
@@ -453,6 +455,49 @@ def frag_size(hit):
     return len(subseq) - subseq.count('-')
 
 
+def correct_header_positions(header, new_start, new_end, out=sys.stderr):
+    """Prevents chaining of start-end positions.
+
+    Parameters
+    ----------
+    header : str
+        The existing fasta header string with start-end positions as a suffix.
```

Comment:
I elaborate on the format in the docstring. It is very customized, i.e. I cannot link to an official format specification.